tools/verify_pr_title.js

```diff
@@ -28,6 +28,8 @@ const validPrefixes = [
   // allow Revert PRs because it allows us to remove the landed
   // commit from the generated changelog
   "Revert ",
+  // allow Reland PRs
```

Comment:
```suggestion
  // allow Reland PRs because when editing the changelog, it helps us identify
  // commits that were reverted, but then relanded
```